database/tables/question_sharing_sets.pg

```diff
@@ -0,0 +1,13 @@
+columns
+    id: bigint not null default nextval('question_sharing_sets_id_seq'::regclass)
+    question_id: bigint not null
+    sharing_set_id: bigint not null
+
+indexes
```

Comment:
should I have multiple indices here, like what `question_tags.pg` shows for the `question_tags` table?

Reply:
done